### ISSUE TITLE
AttributeError: 'PointCloud' object has no attribute 'labels'

### ISSUE THREAD

**Issue by 92shehab** [menpo/menpo#854]
with menpo tutorial code i got an error ! 
link : https://nbviewer.jupyter.org/github/menpo/menpo-notebooks/blob/master/menpo/Landmarks/Basics.ipynb

![Screenshot_2021-03-03 landmark - Jupyter Notebook](https://user-images.githubusercontent.com/62415452/109765513-2a461b00-7c1f-11eb-9e72-501d2d076256.png)

 How to solve it ?

**Comment by patricksnape** [comment]
Yes the menpo tutorials are likely quite out of date with the current package - would love some help updating them if you are going through them!
